admin/class-perfecty-push-admin.php

```diff
@@ -362,10 +369,32 @@ public function on_transition_post_status( $new_status, $old_status, $post ) {
 			$send_notification = ! empty( get_post_meta( $post->ID, '_perfecty_push_send_on_publish', true ) );
 		}
 
+		$send_featured_img = false;
```

Comment:
We could use the nonce validation once and do:

```php
		$send_notification = false;
		$send_featured_img = false;
		$notification_title = '';
		if ( isset( $_POST['perfecty_push_post_metabox_nonce'] ) &&
			wp_verify_nonce( $_POST['perfecty_push_post_metabox_nonce'], 'perfecty_push_post_metabox' ) ) {
			// we do this because on_transition_post_status is triggered before on_save_post by WordPress
			$send_notification = ! empty( $_POST['perfecty_push_send_on_publish'] );
			$send_featured_img = ! empty( $_POST['perfecty_push_send_featured_img'] );
			$notification_title =  $_POST['perfecty_push_notification_custom_title'];
		} else {
			$send_notification = ! empty( get_post_meta( $post->ID, '_perfecty_push_send_on_publish', true ) );
			$send_featured_img = ! empty( get_post_meta( $post->ID, '_perfecty_push_send_featured_img', true ) );
			$notification_title = get_post_meta( $post->ID, '_perfecty_push_notification_custom_title', true );
		}
```